src/System.Drawing.Common/src/System/Drawing/Font.cs

```diff
@@ -14,6 +14,9 @@ namespace System.Drawing
     /// <summary>
     /// Defines a particular format for text, including font face, size, and style attributes.
     /// </summary>
+#if netcoreapp
+    [TypeConverter("System.Drawing.FontConverter, System.Windows.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51")]
```

Comment:
I believe APICompat should make you add these to the reference assembly as well.  We need them for scenarios like https://github.com/dotnet/corefx/issues/32602.  I don't see that failing your build though which is a bit concerning.

Reply:
@ericstj Does it mean anytning to to me and this PR?